docs/src/Groups/groups.md

```diff
@@ -22,6 +22,160 @@ Julia supports the following types of groups:
 * `DirectProductOfGroups` = direct product of two groups
 * `AutomorphismGroup` = group of automorphisms over a group
 
-## Permutations groups
 
-In Julia, permutations are displayed as products of disjoint cycles, as in GAP.
+If `x` is an element of the group `G` of type `T`, then the type of `x` is `GAPGroupElement{T}`.
+
+## Basics
+
+### Elements of groups
+
+Given a group `G`, it is always possible to have access to some particular elements.
+
+```@docs
+one(x::GAPGroup)
+rand(::GAPGroup)
+rand_pseudo(G::GAPGroup)
+gens(::GAPGroup)
+```
+
+If the group `G` has been created as the subgroup of another group generated by a list of elements `L`, then the generating set returned by the function `gens` corresponds to `L`.
+
+!!! note
+    The output of `gens(G)` is not, in general, the minimal list of generators for `G`.
+
+### Operations
+
+Oscar supports the following operations and functions on group elements.
+
+* `*` = multiplication between two elements in a group.
+* `inv(x)` = `x^-1` the inverse of `x`.
+* `x/y` = the element `x*y^-1`.
+* `x^n` = the ``n``-th power of `x`. If ``n = 0``, the identity of the group is returned; if ``n < 0 ``, the ``n``-th power of the inverse of `x` is returned.
+* `isone(x)`: returns whether `x` is the identity of the group.
+* `conj(x,y)` = `x^y` = the conjugate of `x` by `y`, i.e. the element `x^-1*y*x`.
+* `comm(x,y)` = the commutator of `x` and `y`, i.e. the element `x^-1*y^-1*x*y`.
+
+
+## Subgroups
+
+The subgroup of a group `G` generated by the elements `x,y,...` are defined by the following instruction:
+
+* `sub(G, [x,y,...])` ;
+* `sub(x,y,...)`.
+
+This function returns two objects: a group `H`, that is the subgroup of `G` generated by the elements `x,y,...`, and the embedding homomorphism of `H` into `G`. The object `H` has the same type of `G`, and it has no memory of the "parent" group `G`: it is an independent group.
+
+  **Examples:**
+```jldoctest
+julia> G = symmetric_group(4); H = sub(G,[cperm([1,2,3]),cperm([2,3,4])]);
+julia> H[1] == alternating_group(4)
+true
+```
+
+
+## Permutation groups
+
+Permutation groups can be defined as symmetric groups, alternating groups or their subgroups.
+
+```@docs
+symmetric_group
+alternating_group
+```
+
+Every permutation group `G` is identified by a degree, that corresponds ideally to the size of the set on which `G` is acting.
```

Comment:
I don't think every permutation group is "identified" by a degree! Rather, every permutation group *in Oscar* has an associated degree.

The use of "ideally" renders this sentences rather vague. It implies that this "ideal" may not be always reached, but gives the reader no clue at all about when that is and so on. Also, is this really "ideal"? Perhaps you meant "usually"? What exactly do you want to convey here?

Perhaps more something like:
```suggestion
In Oscar, every group has a parent (see section XYZ for details). For permutation groups, the parent currently always is a symmetric group. The degree `d` of a permutation group `G` then is the degree of its parent symmetric group. That is, `G` acts on the set `1..d`.
```

Reply:
Isn't `parent` defined only for group elements? The symmetric group is only a kind of "sup-group" for the permutation group `G`. Maybe we should say something like:
" Every permutation group `G` has a degree `n`, that is the cardinality of the set on which `G` acts."
Then add the note about `deg(G)` != largest moved point of `G` in general.